examples_gbs/run_tutorial_points.py

```diff
@@ -0,0 +1,109 @@
+"""
+Point Process Tutorial
+======================
+
+This tutorial shows how to generate GBS point process samples and use them to detect outlier
+points in a data set. Point processes are models for generating random point patterns and GBS
+devices can be programmed to operate as special types of point processes that generate clustered
```

Comment:
```suggestion
devices can be programmed to operate as generators of a special type of point process that favours sampling clustered point patterns.
```

Maybe start a new paragraph with "GBS point processes..."